seed/challenges/object-oriented-and-functional-programming.json

```diff
@@ -198,6 +198,9 @@
         "// Only change code above this line.",
         "(function() {return array;})();"
       ],
+      "MDNlinks": [
```

Comment:
Almost, but not quite. The `'l` in `links` needs to be upper case. Look at it like a great opportunity to learn about `git commit --amend`.